pages/api/mock/[...slug].ts

```diff
@@ -207,17 +208,19 @@ export default async function handler(
         ]
       : null;
 
-  // TODO: properties can be set as array or object by the schema, currently not
-  // dealing with this
   const properties = schema ? propertiesFromSchema(schema) : null;
 
+  // Just deal with the first layer of nesting for now
+  const isArray = schema?.type === "array";
```

Comment:
What actually happens if we are mocking an array of arrays?

Reply:
updated the comment